distribution/engine/THIRD-PARTY/NOTICE

```diff
@@ -491,11 +306,6 @@ The license file can be found at `licenses/Universal_Permissive_License__Version
 Copyright notices related to this dependency can be found in the directory `org.graalvm.shadowed.xz-24.0.0`.
 
 
-'chromeinspector-tool', licensed under the GNU General Public License, version 2, with the Classpath Exception, is distributed with the engine.
-The license file can be found at `licenses/GNU_General_Public_License__version_2__with_the_Classpath_Exception`.
-Copyright notices related to this dependency can be found in the directory `org.graalvm.tools.chromeinspector-tool-24.0.0`.
```

Comment:
Aren't we distributing it, just in a different way?

Reply:
Yes, as `component/chromeinspector-tool-24.0.0.jar`. Seems like license review tool is unable to determine that. Let's not deal with it in this PR, we need to merge it ASAP.

Reply:
We already have a way of adding such 'implicit' dependencies, see `runtime-and-langs`.

I'm worried that if we don't address it now, we will forget about it. Let's at least create a ticket and try to do a follow up sooner than later.